### PR TITLE
mingw-w64-julia: Switch install to /opt and update to version 1.8.3

### PR DESCRIPTION
**Pull request by stahta01**
Change design to prefer Binary Build tars from Julia instead of
MSys2 MinGW packages. This is the Julia perfered build method; but, 
it causes file conflicts. Because of the file conflicts, the package
is installed into "/opt/julia${MINGW_PREFIX}"
with a julie script placed into ${MINGW_PREFIX}/bin.

This method should result in it being easier to upgrade versions.

The Julia versions 1.8.4, 1.8.5, 1.9.x, and git master has the 
msys2+mingw64 build bug https://github.com/JuliaLang/julia/issues/48081

Please give me feedback on whether the design changes like installing into /opt folder is okay.

Tim S.

### CODE REVIEW

File: mingw-w64-julia/PKGBUILD [hunk 1]
```diff
@@ -25,60 +25,42 @@ _show_log_lockup_info=0   # 0 or 1    (PR value is 0)
...
-         "${MINGW_PACKAGE_PREFIX}-suitesparse"
-         "${MINGW_PACKAGE_PREFIX}-lapack"
-         "${MINGW_PACKAGE_PREFIX}-openblas"
-         "${MINGW_PACKAGE_PREFIX}-libunwind")
-makedepends=("patch"
-             "make"
-             "tar"
-             "${MINGW_PACKAGE_PREFIX}-gcc-fortran"
-             "${MINGW_PACKAGE_PREFIX}-gmp"
+makedepends=('make' 'patch' 'perl' 'tar'
```
Comment (Biswa96) [inline mingw-w64-julia/PKGBUILD:37]: These are already part of base-devel group and not required to add in makedepends.

Comment (stahta01) [reply mingw-w64-julia/PKGBUILD:37]: Okay, are you sure about 'perl' ?

Tim S.

**Comment by stahta01** [comment]
I have made a tiny bit of progress building UCRT64; it made it to 22% of LLVM and it is still building.
I expect to fail before the external build script finishes. But, getting it to start building LLVM was a major mile stone.
Once, the external script builds a working copy of Julia; I will move the changes to the package in this PR.

Tim S.

**Comment by stahta01** [comment]
Someone else did a PR; so, let them worry about this issue.

**Comment by stahta01** [comment]
I accidently closed this PR instead of another one. I am making slow progress on getting UCRT64 to build.

**Comment by stahta01** [comment]
I have failed to get by the Symbolic links with tar command issue with UCRT64 using the standard MINGW packaging method.
I am now trying building with /usr/bin at front of path; once that fails or succeeds I will finish editing this package.
The Symbolic links with tar command issue might be an MSYS2 auto path correction issue, instead. 

Tim S.

**Comment by stahta01** [comment]
I have given up getting UCRT64 to build with this PR. Please someone try it or I will request this package to be removed.

**Pull request closed without merging**